src/components/account-dropdown/account-dropdown-item/account-dropdown-item.tsx

```diff
@@ -23,17 +24,18 @@ export const AccountDropdownItem: FC<AccountDropdownItemProps> = ({
   actionIconName
 }) => {
   const styles = useAccountDropdownItemStyles();
+  const { publicKeyHash, name } = account;
 
   return (
     <View style={styles.root}>
-      <RobotIcon seed={account.publicKeyHash} />
+      <RobotIcon seed={publicKeyHash} />
       <View style={styles.infoContainer}>
         <View style={[styles.upperContainer, conditionalStyle(showFullData, styles.upperContainerFullData)]}>
-          <Text {...getTruncatedProps(styles.name)}>{account.name}</Text>
+          <Text {...getTruncatedProps(styles.name)}>{name}</Text>
           {isDefined(actionIconName) && <Icon name={actionIconName} size={formatSize(24)} />}
         </View>
         <View style={styles.lowerContainer}>
-          <WalletAddress publicKeyHash={account.publicKeyHash} />
+          {publicKeyHash !== EMPTY_PUBLIC_KEY_HASH && <WalletAddress publicKeyHash={publicKeyHash} />}
```

Comment:
should this fix be applied for each place where `WalletAddress` is used?  
mb move this fix inside `WalletAddress` component?